src/components/ProfileInformation/ProfileInformation.tsx

```diff
@@ -200,6 +160,54 @@ const ProfileInformation = (props: Props) => {
           </div>
         )}
       </div>
+    ),
+    []
```

Comment:
Remember to add the variables the function depend on here.
Things like `avatar`, `profileAddress`, `avatarName` and others are important and should be here.